core/src/com/unciv/ui/utils/AudioExceptionHelper.kt

```diff
@@ -0,0 +1,10 @@
+package com.unciv.ui.utils
+
+import com.badlogic.gdx.audio.Music
+
+interface AudioExceptionHelper {
+    fun installHooks(
+        updateCallback: (()->Unit)?,
+        exceptionHandler: ((Throwable, Music)->Unit)?
+    )
+}
```

Comment:
Would it make more sense to place this file in `/ui/audio/` instead of `/ui/utils/`?

Reply:
True. I was halfway between pre-reorg and the idea all platform-dependent-helpers need to be grouped into their own package (still well distributed) - there's a word in psy for it - ya know, a predator threatens a cock potentially defending his hens, and he can't decide whether to attack or run, so he starts picking grains instead?